admin/src/templates/message_queue_tab.html

```diff
@@ -0,0 +1,46 @@
+<p ng-show="error" class="alert alert-danger" role="alert" translate>admin.message.queue.error</p>
+
+<div class="row flex-row selection-heading">
+  <div class="col col-fixed" translate>admin.message.queue.recipient</div>
+  <div class="col col-fixed" translate>admin.message.queue.created</div>
+  <div class="col col-grow" translate>admin.message.queue.message</div>
+  <div class="col col-fixed" translate>admin.message.queue.status</div>
+  <div class="col col-fixed" translate>admin.message.queue.due</div>
+  <div ng-if="displayLastUpdated" class="col col-fixed" translate>admin.message.queue.updated</div>
+</div>
+
+<div class="loader" ng-show="loading"></div>
+
+<div class="row flex-row selection" ng-repeat="message in messages">
+
+  <div class="col col-fixed">{{message.recipient}}</div>
+
+  <div class="col col-fixed">
+    <relative-date date="message.record.reportedDate" date-format="dateFormat"></relative-date>
+    <div ng-if="message.link">
+      <a href=" ../../medic/_rewrite/#/reports/{{message.record.id}}" target="_blank">
```

Comment:
I think we need to find an absolute way to refer to the other app to avoid [issues like this](https://github.com/medic/medic-webapp/issues/4829). The Location service [returns the absolute path](https://github.com/medic/medic-webapp/blob/master/webapp/src/js/services/location.js#L14) to the webapp to get you started.

Reply:
Also TIL it's safer to use [`ng-href`](https://docs.angularjs.org/api/ng/directive/ngHref) to make sure the href property doesn't exist before the expression is evaluated.